x-pack/plugins/task_manager/server/task.ts

```diff
@@ -257,6 +257,11 @@ export interface TaskInstance {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   state: Record<string, any>;
 
+  /**
+   * The serialized traceparent string of the current APM transaction or span.
+   */
+  traceparent?: string;
+
```

Comment:
I think we should add a note in the Readme.md about using this field to trace the execution of tasks in apm.
Other plugins might want to utilise this.